light-node/Cargo.toml

```diff
@@ -7,11 +7,18 @@ publish = false
 
 [dependencies]
 abscissa_tokio = "0.5"
+anomaly = { version = "0.2", features = [ "serializer" ] }
```

Comment:
@tarcieri @tony-iqlusion Mind chiming in here? It's been a while since I actively used all 3 dependencies: `anomaly`, `thiserror`, and `abscissa`. Does it make sense to mix those here? Doesn't make abscissa using anomaly in the same crate obsolete? I'm a bit lost here.

Reply:
Error handling in Abscissa is in the middle of an overhaul to switch to [`eyre`](https://github.com/yaahc/eyre) (or if the improvements in `eyre` end up being upstreamable, potentially [`anyhow`](https://github.com/dtolnay/anyhow)) /cc @yaahc

Reply:
Good to know thanks!

Reply:
It looks like there's a decent chance that the changes will get upstreamed back into `anyhow`, not sure how long it will be though.